docs/en/ingest-management/elastic-agent/configuration/outputs/output-logstash.asciidoc

```diff
@@ -33,27 +33,38 @@ The {ls} configuration pipeline listens for incoming {agent} connections,
 processes received events, and then sends the events to {es}.
 
 The following example configures a {ls} pipeline that listens on port `5044` for
-incoming {agent} connections and routes received events to {es}:
+incoming {agent} connections and routes received events to {es}.
+
+The {ls} pipeline definition below is an example.
```

Comment:
```suggestion
```
I think I would remove this line and add `{ls} pipeline definition into the sentence at line 35 instead.